lib/repl.js

```diff
@@ -59,9 +59,11 @@ const {
   ArrayPrototypeSort,
   ArrayPrototypeSplice,
   ArrayPrototypeUnshift,
+  ArrayPrototypeAt,
```

Comment:
Can you move this line between line 45 and line 46 please?

Reply:
Well, this PR is now blocked, until we have `ArrayPrototypeAt` or I just use the `length` attribute to fetch the last element.

Reply:
> until we have `ArrayPrototypeAt`

What do you mean? We already have it.

Reply:
I don't find `ArrayPrototypeAt` in `typings/primordials.d.ts`

Reply:
I guess that means the typing file is outdated. That doesn't mean it's not there or that you can't use it though.

Reply:
Can you move this line between line 45 and line 46 please?

Reply:
@aduh95 Where else is it defined? Also, does the lint fix reorder the import statements?

Reply:
It's defined by V8 https://github.com/nodejs/node/blob/f226350fcbebd4449fb0034fdaffa147e4de28ea/deps/v8/src/init/bootstrapper.cc#L1792-L1793



> Also, does the lint fix reorder the import statements?

No you need to do that manually.